README.md

```diff
@@ -387,12 +409,18 @@ yarn format-package --help
 6.  [`format-package.config.yaml` or `format-package.config.yml`](#with-format-packageymlyaml-format-packageconfigymlyaml)
 7.  [`format-package`](#with-packagejson) property in `package.json`
 
-If there are no valid configuration, it will use the [default](lib/defaults/index.js).
+If there are no valid configuration from the above search places, move up one directory level and try again.
+So the search continues in ./, ../, ../../, ../../../, etc., checking the same places in each directory.
```

Comment:
We can probably simplify this just a bit and still communicate the point. Also, let's wrap the "./" in back ticks, e.g. `./`

Reply:
Done.